app/javascript/dashboard/components/widgets/WootWriter/ReplyBottomPanel.vue

```diff
@@ -321,17 +340,12 @@ export default {
 .right-wrap {
   display: flex;
 
-  .enter-to-send--checkbox {
-    align-items: center;
-    display: flex;
-
-    input {
-      margin: 0;
-    }
+  .announcement-popup {
```

Comment:
@iamsivin The `AnnouncementPopup` component is displaying in the viewport by calculating the position of enter button. Can we make this component as floating component so that position themselves perfectly? ?

cc @pranavrajs @nithindavid

Reply:
If it makes it generic, that is better.